nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -304,18 +332,23 @@ class TfOpIdxMap {
     }
   }
 
-  template <typename T> BidirectionalIndexMap<T> &getIndexMap() {
-    return std::get<BidirectionalIndexMap<T>>(maps);
+  template <typename KeyType, typename DataType = KeyType>
+  BidirectionalIndexMap<KeyType, DataType> &getIndexMap() {
+    return std::get<BidirectionalIndexMap<KeyType, DataType>>(maps);
   }
 
-  template <typename T> const BidirectionalIndexMap<T> &getIndexMap() const {
-    return std::get<BidirectionalIndexMap<T>>(maps);
+  template <typename KeyType, typename DataType = KeyType>
+  const BidirectionalIndexMap<KeyType, DataType> &getIndexMap() const {
+    return std::get<BidirectionalIndexMap<KeyType, DataType>>(maps);
   }
 
+  const float *get_empty_buffer() const { return empty_buffer; }
+
 private:
-  float empty_buffer[0]; /**< unintialized tensor points to this buffer */
+  float
+    empty_buffer[0]; /**< reserved unintialized tensor points to this buffer */
 
-  std::tuple<BidirectionalIndexMap<const float *>, /**< underlying buffer map */
+  std::tuple<BidirectionalIndexMap<const float *, Buffer>,   /**< buffer map */
```

Comment:
I wonder, Would it be more as we support more layer?

Reply:
From spec, I think there wouldn't be no new map.